README.md

```diff
@@ -25,6 +25,33 @@ are not required when you don't want to use the WordPress image upload with S3.
 
 In reverse, if you activate the already preinstalled S3 plugin you must specify these variables.
 
+By default, the default AWS endpoint (s3.amazonaws.com) will be used.
+Be careful of that if you're using another provider.
+
+### Use another S3 provider
+
+If you want to use another provider than AWS for your S3 bucket.
+You may want to use the environment variable named `S3_UPLOADS_ENDPOINT` to specify your provider URL.
```

Comment:
This is the same sentence.
```suggestion
If you want to use another provider than AWS for your S3 bucket,
you may want to use the environment variable named `S3_UPLOADS_ENDPOINT` to specify your provider URL.
```

Reply:
You write "you may want to use...". But isn't it mandatory if one use a provider different than S3 to set the env variable `S3_UPLOADS_ENDPOINT`?